Admin/FieldDescription.php

```diff
@@ -16,7 +16,7 @@
 class FieldDescription extends BaseFieldDescription
 {
     /**
-     * {@inheritdoc}
+     * Constructor.
```

Comment:
I opened a PR for the doc to match what is done on the repo : https://github.com/symfony/symfony-docs/pull/5838